website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+#### Decay Module
+
+This module calculates the annual decay and turnover on a set of dead organic matter pools present in the GCBM. The data requirements for the Decay Module are:
+    
+- A table named "decay_parameters", with one set of decay parameters for each of the enumerated dom pools in the DomPool.
+
+- Scalar "mean_annual_temperature". This is the mean annual temperature of the environment.
+
+- Scalar "SlowMixingRate". This is the amount that has moved from slow aboveground to slow belowground annually.
+
+#### Moss Decay Module
+
+This module is used for moss related computing. It determines whether the moss pool is decaying or not. It uses various mathematical equations to apply decay rates to the different moss pools.
+
+#### Moss Disturbance Module
+
+This module responds to the fire disturbance events in CBM. This module is responsible for transferring carbon content from moss pools to the GHG pools. It gets the input data from the variable: ```moss_fire_parameters```.
+
+#### Moss Growth Module
+
+This module is responsible for the growth of moss pools. It increments the moss growth with the help of mathematical equations, various parameters and moss pools.
+
+#### Moss Turnover Module
+
+This module is responsible for the turnover of moss pools. The movement between the moss live pool and the moss past pool. It accurately calculates the amount of turnover moss by taking moss live amount and transfer rates into consideration and by applying transfer events.
+
+#### Peatland after CBM Module
+
+This module is triggered after CBM simulation on a land unit, and prepares the land unit to simulate the peatland simulation. It transfers carbon from some CBM pools to the peatland pools. It is called after finishing the regular CBM simulation.
+
+#### Peatland Decay Module
+
+This module is responsible for peatland decay. It uses the variables: ```peatland_decay_parameters``` and ```peatland_turnover_parameters```. It also uses various parameters like: mean annual temperature, annual water table depth, total initial carbon. It uses these to transfer the pool's content.
+
+It also uses a special turnover rate for various pools like carotelm pool. It sets the turnover rate for different pools by taking into account the values of various GHG.
+
+#### Peatland Disturbance Module
+
+This module responds to the historical and last disturbance events in the CBM spinup. It alters the water table depth according to the disturbance type and Peatland Id.
+
+#### Peatland Growth Module
+
+This module gets the data from the variables: ```peatland_growth_parameters```, ```peatland_turnover_parameters``` and ```peatland_growth_curve```. It simulates the various growth cycles, growth curves and also sets the component value for the pools.  For example: woody layer, moss layer and many more.
```

Comment:
It would be good to describe all of the pools; do you know how many more there are?

Reply:
Nope. Can you give a background on this?